dev-util/r2dec_js/r2dec_js-5.9.8.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="Pseudo-code decompiler for radare2"
+DESCRIPTION="r2dec-js, a radare2 plugin - converts asm to pseudo-C code"
+HOMEPAGE="https://github.com/wargio/r2dec-js/"
+COPYRIGHT="2020-2023 Giovanni Dante Grazioli
+	2018-2019 elicn
+	2017-2024 Fabrice Bellard
+	2017-2024 Charlie Gordon
+	2023-2025 Ben Noordhuis
+	2023-2025 Saúl Ibarra Corretgé
+	1998-2019 Toni Ronkko
+	2023 Marcin Kolny"
+LICENSE="BSD (3-clause)
+	MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/wargio/r2dec-js/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="ff8694f7ed5ced04c5119545b06adcc089393c05154e35c0bdc2b9fb43e052c3"
+SOURCE_DIR="r2dec-js-$portVersion"
+quickjsVer="0.8.0"
+SOURCE_URI_2="https://github.com/quickjs-ng/quickjs/archive/refs/tags/v$quickjsVer.tar.gz"
+CHECKSUM_SHA256_2="7e60e1e0dcd07d25664331308a2f4aee2a88d60d85896e828d25df7c3d40204e"
+SOURCE_DIR_2="quickjs-$quickjsVer"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+radare2Ver="5.9.8"
+
+PROVIDES="
+	r2dec_js$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	devel:libr_config$secondaryArchSuffix
+	devel:libr_cons$secondaryArchSuffix
+	devel:libr_core$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libr_config$secondaryArchSuffix == $radare2Ver
+	devel:libr_cons$secondaryArchSuffix == $radare2Ver
+	devel:libr_core$secondaryArchSuffix == $radare2Ver
+	devel:libr_io$secondaryArchSuffix == $radare2Ver
+	devel:libr_util$secondaryArchSuffix == $radare2Ver
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:meson
+	cmd:ninja
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p subprojects/libquickjs
+	cp -R $sourceDir2/* subprojects/libquickjs
+	cp subprojects/packagefiles/libquickjs/meson.build subprojects/libquickjs
+
+	meson setup build \
+		--buildtype=release \
+		--prefix=$prefix \
+		-Dr2_plugdir=$libDir/radare2/$radare2Ver
+
+	ninja -C build $jobArgs
+}
+
+INSTALL()
+{
+	ninja -C build install
+}
+
+TEST()
+{
+	make check
```

Comment:
This is wrong. This doesn't use make to build. I guess this should have been `ninja -C build test`.

Reply:
You're right, seems I didn't do my work here ;) Adding the test fails with meson not found now :/

Reply:
Fixed in https://github.com/haikuports/haikuports/pull/11827